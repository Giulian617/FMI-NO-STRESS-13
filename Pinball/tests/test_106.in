100 279321939710820590
4 1 2
1 43 3
29 1 3
87 1 9
42 1 4
1 36 6
81 1 10
1 93 1
1 55 9
28 1 9
53 1 8
1 32 6
1 100 2
14 1 6
84 1 10
1 64 4
90 1 10
48 1 6
77 1 8
1 75 1
71 1 1
1 39 3
1 38 3
1 11 3
1 97 2
1 49 9
23 1 4
61 1 7
57 1 4
95 1 4
1 17 7
1 30 9
58 1 7
1 10 10
22 1 10
1 74 2
1 12 7
25 1 8
69 1 10
1 68 4
7 1 3
33 1 6
59 1 9
45 1 3
1 92 7
1 63 1
78 1 7
5 1 4
1 20 7
1 72 1
1 13 9
50 1 8
2 1 7
98 1 10
1 88 2
1 19 8
1 18 1
15 1 9
54 1 1
41 1 6
1 94 4
1 31 10
83 1 1
9 1 2
26 1 9
73 1 9
1 52 1
85 1 9
21 1 10
1 37 6
3 1 10
1 76 6
82 1 3
56 1 3
96 1 5
1 16 3
66 1 9
51 1 9
67 1 9
80 1 10
8 1 2
89 1 2
1 62 2
1 70 6
1 46 5
1 86 6
1 24 3
6 1 8
1 91 10
79 1 2
1 47 5
1 99 3
65 1 1
27 1 3
1 35 2
40 1 1
44 1 5
60 1 3
1 34 3
