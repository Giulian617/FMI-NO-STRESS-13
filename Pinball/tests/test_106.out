290127018
222098258
869306955
51234202
894153968
998684251
869306955
869306955
222098258
998684251
290127018
741268266
72419074
741268266
998684251
290127018
612456720
894153968
290127018
222098258
222098258
51234202
998684251
894153968
741268266
998684251
741268266
998684251
741268266
222098258
72419074
72419074
998684251
869306955
72419074
72419074
998684251
998684251
612456720
72419074
51234202
998684251
464349267
998684251
464349267
464349267
72419074
741268266
894153968
741268266
612456720
894153968
612456720
741268266
998684251
51234202
290127018
741268266
998684251
290127018
869306955
612456720
51234202
612456720
741268266
741268266
51234202
222098258
72419074
612456720
612456720
741268266
869306955
612456720
72419074
894153968
290127018
869306955
222098258
222098258
998684251
612456720
222098258
741268266
72419074
741268266
869306955
869306955
222098258
222098258
290127018
612456720
51234202
51234202
464349267
869306955
222098258
998684251
869306955
